ci: master branch ci failed

https://github.com/apache/apisix/runs/2175788179#step:15:823
https://github.com/apache/apisix/runs/2175041905#step:15:996

```
Test Summary Report
-------------------
t/discovery/dns/sanity.t                 (Wstat: 512 Tests: 42 Failed: 2)
  Failed tests:  41-42
  Non-zero exit status: 2
Files=207, Tests=8274, 1288 wallclock secs ( 2.08 usr  0.31 sys + 197.84 cusr 80.01 csys = 280.24 CPU)
Result: FAIL
Error: Process completed with exit code 1.
```

--- a/apisix/core/dns/client.lua
+++ b/apisix/core/dns/client.lua
@@ -73,6 +73,7 @@ local function resolve_srv(client, answers)
             local copy = table.deepcopy(res)
             copy.weight = weight / count
             copy.port = answer.port
+            copy.priority = answer.priority
             insert_tab(resolved_answers, copy)
         end
     end

